File: Beginner/Python/Odd_or_not.py
# github username : Titan1911

# program to check an odd number

print('**********Welcome to the program to check whether a number is odd or not**********')
check_number = int(input('Enter a positive number : '))
if check_number>=0:
    if check_number%2==0:
        print('The number is not odd')
    if check_number%2==1:
        print('The number is an odd number')

else:
    print("Invalid Number")

File: Beginner/Java/odd_or_not.java
import java.util.Scanner;
public class odd_or_not {
 public static void main(String[] args) {
   Scanner sc = new Scanner(System.in);
     System.out.println("Enter a number : ");
     int n = sc.nextInt();
     if(n%2==0)
     {
         System.out.println("Number is not odd.");
     }
     else
     {
         System.out.println("Number is Odd.");
     }
 }
}
